resources/assets/coffee/react/profile-page/extra.coffee

```diff
@@ -90,20 +126,51 @@ class ProfilePage.Extra extends React.Component
 
   _removeListeners: ->
     $.unsubscribe '.profileContentsExtra'
+    $.unsubscribe 'profilePageExtraToggle:toggleDragging'
     $(window).off '.profileContentsExtra'
 
 
   _tabsStick: (_e, target) =>
     @setState tabsSticky: (target == 'profile-extra-tabs')
 
+  _toggleDragging: =>
+    if @state.draggingEnabled
+
+      csrfParam = $('meta[name=csrf-param]').attr('content')
```

Comment:
(also this variable isn't actually used because you did it wrong (should be `"#{csrfParam}": csrfToken` instead of `csrfParam: csrfToken`)